test/src/e2e/partialSignature.test.ts

```diff
@@ -18,16 +18,51 @@ import * as chai from "chai";
 import * as chaiAsPromised from "chai-as-promised";
 chai.use(chaiAsPromised);
 const expect = chai.expect;
-import { Asset, AssetAddress, H256 } from "codechain-sdk/lib/core/classes";
+import {
+    Asset,
+    AssetAddress,
+    H256,
+    Transaction,
+    SignedTransaction
+} from "codechain-sdk/lib/core/classes";
 import * as _ from "lodash";
 import "mocha";
 import { faucetAddress } from "../helper/constants";
 import CodeChain from "../helper/spawn";
+import { AssetTransaction } from "codechain-sdk/lib/core/Transaction";
+import { ChainRpc } from "codechain-sdk/lib/rpc/chain";
+import { Key } from "codechain-sdk/lib/key";
+
+// If one only sends certainly failing trasactions, the miner would not generate any block.
+// So to clearly check the result failed, insert the failing transactions inbetween succeessful ones.
+async function sendFailingTx(
+    node: CodeChain,
+    targetTx: Transaction & AssetTransaction
+): Promise<H256> {
+    await node.sdk.rpc.devel.stopSealing();
+
+    const blockNumber = await node.getBestBlockNumber();
+    const seq = await node.sdk.rpc.chain.getSeq(faucetAddress);
+    const signedDummyTx = await node.sendPayTx({ seq, quantity: 1 });
+    const targetTxHash = await node.sendAssetTransaction(targetTx, {
+        seq: seq + 1
+    });
+
+    await node.sdk.rpc.devel.startSealing();
+    await node.waitBlockNumber(blockNumber + 1);
+
+    expect(await node.sdk.rpc.chain.containsTransaction(signedDummyTx.hash()))
+        .be.true;
+    return targetTxHash;
+}
 
 describe("Partial signature", function() {
     let node: CodeChain;
+    let key: Key;
```

Comment:
The same comment as https://github.com/CodeChain-io/codechain/pull/1475#discussion_r275603525

Reply:
ping?

Reply:
I've reverted this change.